### PR TITLE
{compiler}[system/system] NVHPC v21.7

### PR DESCRIPTION
**Pull request by robert-mijakovic**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by robert-mijakovic** [comment]
@boegelbot please test @ generoso

**Comment by robert-mijakovic** [comment]
Test report by @robert-mijakovic
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
login01.meluxina.lxp.lu - Linux RHEL 8.3, x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/55fa88fe48855c1622bab422493bddcb for a full test report.

**Comment by robert-mijakovic** [comment]
`== 2021-07-23 17:05:54,227 build_log.py:169 ERROR EasyBuild crashed with an error (at easybuild/easybuild-framework/easybuild/base/exceptions.py:124 in __init__): A default Compute Capability is needed for installation of NVHPC.Please provide it either in the easyconfig file like 'cuda_compute_capabilities="7.0"',or use 'eb --cuda-compute-capabilities=7.0' from the command line. (at easybuild/easybuild-easyblocks/easybuild/easyblocks/n/nvhpc.py:132 in install_step)`

**Comment by robert-mijakovic** [comment]
@boegelbot please test @ generoso EB_ARGS='cuda_compute_capabilities="7.0"'

File: easybuild/easyconfigs/n/NVHPC/NVHPC-21.7.eb [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+name = 'NVHPC'
+version = '21.7'
+
+homepage = 'https://developer.nvidia.com/hpc-sdk/'
+description = """C, C++ and Fortran compilers included with the NVIDIA HPC SDK (previously: PGI)"""
+
+toolchain = SYSTEM
+
+# By downloading, you accept the HPC SDK Software License Agreement (https://docs.nvidia.com/hpc-sdk/eula/index.html)
+accept_eula = True
```
Comment (lexming) [inline easybuild/easyconfigs/n/NVHPC/NVHPC-21.7.eb:13]: The EULA should not be accepted by default
```suggestion
# By downloading, you accept the HPC SDK Software License Agreement
# https://docs.nvidia.com/hpc-sdk/eula/index.html
# accept_eula = True
```

**Review by lexming — CHANGES_REQUESTED** [review]

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node226.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v4 @ 2.40GHz, Python 2.7.5
See https://gist.github.com/e35fc484432ac0bc7e5d4db5e5945a40 for a full test report.

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @robert-mijakovic !

**Pull request merged**